src/mlpack/methods/ann/ffn.hpp

```diff
@@ -88,6 +88,23 @@ class FFN
       OutputLayerType&& outputLayer = OutputLayerType(),
       InitializationRuleType initializeRule = InitializationRuleType());
 
+  /**
+   * Create the FFN object with the given predictors and responses set (this is
+   * the set that is used to train the network) and the given optimizer.
+   * Optionally, specify which initialize rule and performance function should
+   * be used.
+   *
+   * @param predictors Input training variables.
+   * @param responses Outputs results from input training variables.
+   * @param outputLayer Output layer used to evaluate the network.
+   * @param initializeRule Optional instantiated InitializationRule object
+   *        for initializing the network parameter.
+   */
+  FFN(arma::mat&& predictors,
+      arma::mat&& responses,
+      OutputLayerType&& outputLayer = OutputLayerType(),
+      InitializationRuleType initializeRule = InitializationRuleType());
```

Comment:
Good point, I can't think of any situation where that might be necessary. Maybe I just looked at the LinearRegression/LogisticRegression class and thought it's a good idea.